x/go/io/fs/fs_test.go

```diff
@@ -436,17 +463,22 @@ var _ = Describe("FS", func() {
 			})
 
 			Describe("Truncate", func() {
-				It("Should not truncate a file without write perms", func() {
-					f, err := fs.Open("a.txt", os.O_CREATE|os.O_RDONLY)
-					Expect(err).ToNot(HaveOccurred())
-					err = f.Truncate(5)
-					if s := MustSucceed(fs.Stat("")); s.Sys() == nil {
-						Expect(err).To(MatchError(ContainSubstring("fs: file was not created for writing (truncate requires write fd)")))
-					} else {
-						Expect(err).To(MatchError(ContainSubstring("invalid argument")))
-					}
-					Expect(f.Close()).To(Succeed())
-				})
+				/* When os.Truncate is called on a Windows file with read only perms,
```

Comment:
Should we just entirely remove this test?

Reply:
Yes good call, I will add a comment on top of `os.Truncate` then to indicate the perm issue with windows